identity/client/src/utility/api/membership/index.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+import { ApiDefinition, apiDelete, apiGet, apiPost } from "../request";
+import { User } from "src/utility/api/users";
+import { GROUPS_ENDPOINT } from "src/utility/api/groups";
+
+export type GetGroupMembersParams = {
+  groupId: string;
+};
+
+export const getGroupMembers: ApiDefinition<User[], GetGroupMembersParams> = ({
+  groupId,
+}) => apiGet(`${GROUPS_ENDPOINT}/${groupId}/users`);
+
+type AssignGroupMemberParams = GetGroupMembersParams & { userId: string };
```

Comment:
It's difficult to check api calls without having the endpoints in place, but we can adjust these when the api is merged